src/docs/product/dashboards/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Dashboards
+sidebar_order: 6
+redirect_from:
+  - /workflow/visibility/
+  - /workflow/dashboards/
+  - /product/error-monitoring/dashboards/
+description: "Sentry's Dashboards provide a single view of your application's health, including the ability to navigate issues and performance across multiple projects."
+---
+
+<Alert title="Note" level="info">
+  Dashboards is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
```

Comment:
```suggestion
  Dashboards are only available if you're in the Early Adopter program. Features available to Early Adopters are still in progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
```